usdoj-crt/crt-portal
# Mark more strings for translation

[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/152)

## What does this change?

+ Mark Primary Issue strings for translation in `model_variables.py` and confirm that those work as expected / do not change what's stored in the database.
+ Mark Contact strings for translation in `forms.py`.
+ Fixes spelling mistake in `model_variables.py` and removes sneaky stray Unicode character that was causing the `makemessages` command to fail.

### What does this NOT change?

+ Used this branch to verify that Django `gettext_lazy` does not work out-of-the box with our Protected Class strings stored in the database; a refactor and/or a library may be required to translate the database strings for Protected Class names. 

## Screenshots (for front-end PR):

See screenshots in code comments below.

## Checklist:

### Author

+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

### How to test this PR 

1. In `settings.py`, change the value of `LANGUAGE_CODE` from `en-us` to `es`.
2. Navigate to the `es` `django.po` file:  `crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po`.
3. Change the value of `msgstr` for one of the strings you'd like to translate related to Contact Form or Primary Issue — supply any test value you like (pseudolocalization testing). 
4. Run `docker-compose run web django-admin compilemessages`.
5. Start the site locally and navigate to a page that contains the translated content to see your fake test translation value.
